src/connection.cpp

```diff
@@ -26,16 +26,13 @@
 #include "scheduler.h"
 #include "server.h"
 
-bool Connection::logError = true;
-
 extern ConfigManager g_config;
 
-Connection_ptr ConnectionManager::createConnection(boost::asio::ip::tcp::socket* socket,
-        boost::asio::io_service& io_service, ServicePort_ptr servicer)
+Connection_ptr ConnectionManager::createConnection(boost::asio::io_service& io_service, ConstServicePort_ptr servicePort)
 {
 	std::lock_guard<std::mutex> lockClass(connectionManagerLock);
-
-	auto connection = std::make_shared<Connection>(socket, io_service, servicer);
+	
```

Comment:
I use this to find traling spaces:
```
[ \t]+$
```

I use notepad++, if you do the same - ctrl+f - find in files:
Find what: [ \t]+$
Replace with: (leave empty, otherwise it will replace things in items.otb aswell as *.otbm.
Directory: path to the folder
Search mode: regular expression